Demos/Screensaver/Screensaver.cpp

```diff
@@ -26,19 +26,35 @@
 
 #include <LibCore/ElapsedTimer.h>
 #include <LibGfx/Bitmap.h>
-#include <LibGUI/Application.h>
 #include <LibGUI/Action.h>
+#include <LibGUI/Application.h>
 #include <LibGUI/Event.h>
 #include <LibGUI/Painter.h>
 #include <LibGUI/Widget.h>
 #include <LibGUI/Window.h>
-#include <math.h>
+#include <time.h>
 #include <stdio.h>
 
 #define WIDTH 64
 #define HEIGHT 48
 #define INTERVAL 10000
 
+/* Random functions...
+ * These are from musl libc's prng/rand.c
+*/
+static uint64_t seed;
+
+void my_srand(unsigned s)
+{
+    seed = s - 1;
+}
+
+static int my_rand(void)
+{
+    seed = 6364136223846793005ULL * seed + 1;
+    return seed >> 33;
+}
```

Comment:
Why not use `srand()` and `rand()` from LibC?

Reply:
> 
> 
> Why not use `srand()` and `rand()` from LibC?

It seemed like a [good idea](https://github.com/SerenityOS/serenity/blob/master/Demos/Fire/Fire.cpp#L73) at the time.